src/styles/theme/palette.ts

```diff
@@ -36,6 +35,7 @@ export const PALETTE: PaletteOptions = {
   },
   divider: '#000',
   custom: {
-    colorAquaBlue: '#73ddff',
+    main: '#73ddff',
```

Comment:
in the future if we have more colors here - it may make sense to give it some meaningful name. But so far it's good with `custom` too.